test/spec/modules/33acrossBidAdapter_spec.js

```diff
@@ -200,6 +200,14 @@ describe('33acrossBidAdapter:', function () {
       return this;
     };
 
+    this.withCoppa = coppaValue => {
```

Comment:
what if you just add the coppa signal when it's present.

e.g.
```
const coppaValue = config.getConfig('coppa');

if (coppaValue !== undefined) {
  ttxRequest.regs.coppa = Number(!!coppaValue);
}
```

wouldn't that help to reduce the `.withCoppa(0)` calls in the specs and also be more in sync with other signals that we add to the request only when they are present such as the us_privacy and gpp. LMK what you think about this suggestion.